generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Resolver.java

```diff
@@ -321,6 +322,14 @@ private String downloadSha1(Rule rule) {
     return null;
   }
 
+  /**
+   * Prints currently downloaded maven artifact to console.
+   */
+  private static void logArtifactDownload(Model model) {
+    System.out.println("\tDownloading maven artifact " + model.getGroupId() + ":"
```

Comment:
Thought about the phrasing and changed my mind :) This isn't actually downloading the artifact (just the pom) so how about "Download pom for "?

Also, could you use the logger, not stdout?